server/@types/viewModels/index.d.ts

```diff
@@ -361,6 +361,36 @@ declare module 'viewModels' {
     level: 'ENTRY_LEVEL' | 'LEVEL_1' | 'LEVEL_2' | 'LEVEL_3' | 'LEVEL_4' | 'LEVEL_5' | 'LEVEL_6' | 'LEVEL_7' | 'LEVEL_8'
   }
 
+  export interface Timeline {
+    reference: string
+    prisonNumber: string
+    events?: Array<TimelineEvent>
+  }
+
+  export interface TimelineEvent {
+    reference: string
+    sourceReference: string
+    eventType:
+      | 'ACTION_PLAN_CREATED'
+      | 'GOAL_CREATED'
+      | 'GOAL_UPDATED'
+      | 'GOAL_STARTED'
+      | 'GOAL_COMPLETED'
+      | 'GOAL_ARCHIVED'
+      | 'STEP_UPDATED'
+      | 'STEP_NOT_STARTED'
+      | 'STEP_STARTED'
+      | 'STEP_COMPLETED'
```

Comment:
As things stand, the business don't want to see changes to individual steps on the Timeline (it's too detailed), so it might be best to remove these.